File: task-03/Go/diamond_file.go
package main
import (
	"fmt"
	"os"
	"strconv"
	"strings"
)
func main() {
	data, _ := os.ReadFile("input.txt")
	output, _ := os.OpenFile("output.txt", os.O_APPEND|os.O_CREATE|os.O_WRONLY)
	defer output.Close()
	num, _ := strconv.Atoi(str(data))
	var stars string
	stars = strings.Repeat(" ", num)
	for i := 0; i < num; i++ {
		stars = stars[1:]
		stars += " *"
		output.WriteString(stars + "\n")
	}
	for j := 0; j < num; j++ {
		stars = stars[:len(stars)-2]
		stars = " " + stars
		output.WriteString(stars + "\n")
	}
}

File: task-03/Javascript/diamond_file.js
const fs = require('fs');
fs.readFile('input.txt', (err, data) => {
    if (err) throw err;
    console.log(data.toString());
    let num = parseInt(data);
    fs.writeFile('output.txt', '', (err) => {
        if (err) throw err;
        let star = " ".repeat(num);
        for (let i = 0; i < num; i++) {
            star = star.slice(1);
            star += " *";
            fs.appendFile('output.txt', star, (err) => {
                if (err) throw err;
            });
        }
        for (let j = 0; j < num; j++) {
            star = star.slice(0, -2);
            star = " " + star;
            fs.appendFile('output.txt', star, (err) => {
                if (err) throw err;
            });
        }

    });
});